src/lib.rs

```diff
@@ -140,15 +140,35 @@ where
     Ok(pk)
 }
 
+#[deprecated(note = "Use encrypt_key_with_id() instead")]
```

Comment:
@tmpfs I don't think we need to deprecate `encrypt_key` since it can be used by users/dependent crates that wish to assign the file name with a random Uuid.

Reply:
Cool, removed the deprecation notice and restored the doc comments 👍